src/test/java/com/commercetools/sync/categories/helpers/CategorySyncStatisticsTest.java

```diff
@@ -0,0 +1,140 @@
+package com.commercetools.sync.categories.helpers;
+
+
+import io.netty.util.internal.StringUtil;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.commercetools.sync.commons.helpers.BaseSyncStatistics.getStatisticsAsJSONString;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CategorySyncStatisticsTest {
+
+    private static CategorySyncStatistics CATEGORY_SYNC_STATISTICS;
```

Comment:
testing a ***static mutable*** value used to be a bad practice! use a class member